lib/srv/desktop/discovery.go

```diff
@@ -130,7 +130,11 @@ func (s *WindowsService) getDesktopsFromLDAP() types.ResourcesWithLabelsMap {
 	filter := s.ldapSearchFilter()
 	s.cfg.Log.Debugf("searching for desktops with LDAP filter %v", filter)
 
-	entries, err := s.lc.readWithFilter(s.cfg.DiscoveryBaseDN, filter, computerAttribtes)
+	var attrs []string
+	attrs = append(attrs, computerAttribtes...)
```

Comment:
```suggestion
	attrs = append(attrs, computerAttributes...)
```

Reply:
🤦‍♂️this has been in the codebase for a long time!